src/covid_shared/ihme_deps.py

```diff
@@ -30,6 +33,19 @@ def f(*args, **kwargs):
     get_location_hierarchy_by_version = _lazy_import_callable('db_queries.api.internal',
                                                               'get_location_hierarchy_by_version')
 
+
+def load_location_hierarchy(location_set_version_id: int = None,
+                            location_file: Path = None):
+    assert ((location_set_version_id and not location_file)
+            or (not location_set_version_id and location_file))
+
+    if location_set_version_id:
+        return get_location_hierarchy_by_version(
```

Comment:
Does this handle the "release_id", "gbd_id" change that just happened to get_location_metadata?

Reply:
This is an internal function in db_queries that only needs lsvid (which is a unique marker for a hierarchy anyway).  Release id and gbd_round_id are about validation wrappers, which we do not care about since we don't work on GBD